### ISSUE TITLE
[Feature request] Generate random password button

### ISSUE THREAD

**Issue by womeier** [Nitrokey/nitrokey-app2#301]
When adding new passwords, the old app has a button to generate random passwords.
I'm missing this in the new app.

thanks!

**Comment by ABuljko** [comment]
@robin-nitrokey 
Can we close this?

**Comment by robin-nitrokey** [comment]
Yes. You can also link your PRs to the issues they’re fixing, see https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue, to close them automatically once they are merged.

**Comment by ABuljko** [comment]
That actually helps a lot TY <3 <3
